<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>1073</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="9" column="3">
       <widget class="QPushButton" name="Plot_Refresh_2">
        <property name="text">
         <string>Refresh Graph</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="12" column="3" rowspan="2">
       <widget class="QPushButton" name="Plot_clear">
        <property name="text">
         <string>Clear Selection Lines</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="4">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Selections</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Delete Family</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="TreeView" name="treeView">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Add Family</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="4" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>List of Materials</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="plot_prop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Plot Property</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QLineEdit" name="x_range_l">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="Exp_Ratio">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="y_range_h">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Expansion ratio</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>x-axis range</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>y-axis range</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="x_range_h">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="y_range_l">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="logRadio">
              <property name="text">
               <string>LogScale</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QRadioButton" name="linearRadio">
              <property name="text">
               <string>Linear</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="more_axis_setting">
              <property name="text">
               <string>More</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="lable_property">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Lable Property</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="3">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Family bubble</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QFontComboBox" name="fontComboBox_MatB">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="Fontsize_MatB">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="Fontsize_FamB">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QFontComboBox" name="fontComboBox_FamB">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Material bubble</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Font size</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="clear_family">
        <property name="text">
         <string>Clear Family Bubble</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="draw_Family">
        <property name="text">
         <string>Generate Family Bubble</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="Plot_Prop_Chrt">
        <property name="text">
         <string>Generate Material Bubble</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Family Categorizing</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="family_column"/>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="Refresh_Tree">
        <property name="text">
         <string>Refresh List</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QPushButton" name="Plot_sel_ln">
        <property name="text">
         <string>Plot Selection Lines</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="AGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>800</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1327</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_CSV"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Material"/>
    <addaction name="actionDelete_Material"/>
    <addaction name="separator"/>
    <addaction name="actionGroup_Family"/>
    <addaction name="actionUngroup_Family"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFeature_Request">
    <property name="title">
     <string>Feature Request</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSelect"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuFeature_Request"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHotReload"/>
   <addaction name="actionConvexHull"/>
   <addaction name="actionResetView"/>
   <addaction name="actionFitView"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_CSV"/>
   <addaction name="actionFamily"/>
   <addaction name="actionAxes"/>
   <addaction name="actionGenerateChart"/>
   <addaction name="actionFamilyBubble"/>
   <addaction name="actionSelection"/>
   <addaction name="actionPlotSelLn"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
  </widget>
  <action name="actionOpen_CSV">
   <property name="text">
    <string>Open CSV...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAxes">
   <property name="text">
    <string>Axes</string>
   </property>
   <property name="toolTip">
    <string>Axes</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAdd_Material">
   <property name="text">
    <string>Add Material</string>
   </property>
  </action>
  <action name="actionDelete_Material">
   <property name="text">
    <string>Delete Material</string>
   </property>
  </action>
  <action name="actionGroup_Family">
   <property name="text">
    <string>Group Family</string>
   </property>
  </action>
  <action name="actionUngroup_Family">
   <property name="text">
    <string>Ungroup Family</string>
   </property>
  </action>
  <action name="actionHotReload">
   <property name="text">
    <string>HotReload</string>
   </property>
   <property name="toolTip">
    <string>Hot reload your code</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionConvexHull">
   <property name="text">
    <string>ConvexHull</string>
   </property>
  </action>
  <action name="actionFamily">
   <property name="text">
    <string>Family</string>
   </property>
  </action>
  <action name="actionGenerateChart">
   <property name="text">
    <string>Generate Chart</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionPlotSelLn">
   <property name="text">
    <string>Plot Selection Lines</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionFamilyBubble">
   <property name="text">
    <string>Family Bubble</string>
   </property>
  </action>
  <action name="actionResetView">
   <property name="text">
    <string>ResetView</string>
   </property>
   <property name="toolTip">
    <string>ResetView</string>
   </property>
  </action>
  <action name="actionFitView">
   <property name="text">
    <string>FitView</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">AGraphicsView</header>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header location="global">TreeView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
